### ISSUE TITLE
Smiley More Box Formatting

### ISSUE THREAD

**Issue by sbulen** [SimpleMachines/SMF#4974]
When configured as 'popup' smileys, they show up in a special window when you press the MORE button.  

This special window has issues, & needs some help on the .css front...  

The MORE box is too big and doesn't fit:
![moreboxtoobig](https://user-images.githubusercontent.com/23568484/44745951-fde53d00-aabd-11e8-9ab4-3994036ce696.JPG)
![moreboxtoobig2](https://user-images.githubusercontent.com/23568484/44745952-fde53d00-aabd-11e8-9e0a-55711788456d.JPG)

Depending on your zoom setting, it disappears altogether:
![moreboxhidden](https://user-images.githubusercontent.com/23568484/44745950-fd4ca680-aabd-11e8-87d9-43f52184881b.JPG)

Also, would it be possible to get the MORE button itself on the same line as the smileys, to the right?  It uses a lot of real estate:
![morecenter](https://user-images.githubusercontent.com/23568484/44745954-fde53d00-aabd-11e8-81a5-0ea5b46b626d.JPG)

**Comment by SychO9** [comment]
Where can I enable this popup ?

**Comment by smfbigguy** [comment]
The "more" box is not supposed to fit in the post box. I'm personally not sure why it shows up like that for you but mine shows up above everything. Oh I didn't see the zoom level comment. Why would you be zooming in on it. 
![firefox_screenshot_2018-08-28t20-22-18 272z](https://user-images.githubusercontent.com/3620111/44748848-cc309e00-aade-11e8-9d9d-756adb65b629.png)

**Comment by SychO9** [comment]
@smfbigguy perhaps you're using an older forum version,
because it shows under the editor for me, there's an 
>overflow: hidden

property to the sce-container, not sure why it was added

**Comment by smfbigguy** [comment]
Hmm, not sure why then. I updated my site on or about the 22cnd of this month. I will do it again soon and check things out. Didn't mean to sound rude if I did. :)

**Comment by sbulen** [comment]
Too much blank space no matter how you look at it.  
It disappears when you zoom OUT a little.  
I tried getting rid of the fixed height & width, and it shrunk to an appropriate size around the content, which was nice, but the CLOSE button looked wonky.   So I figured I'd leave this to someone more familiar with the quirks of .css to make it purdy.  

On an SMF-weighted rudeness scale, that didn't even register...
